Sprint-3/implement/get-card-value.js

```diff
@@ -29,3 +29,22 @@
 // Given a card with an invalid rank (neither a number nor a recognized face card),
 // When the function is called with such a card,
 // Then it should throw an error indicating "Invalid card rank."
+
+function getCardValue(card) {
+  let rank = card.slice(0, -1);
+  if (parseInt(rank) >= 2 && parseInt(rank) <= 10) {
+    return parseInt(rank);
```

Comment:
You're calling `parseInt` on the same input here three times - can you think how you can avoid needing to repeat this? Why is doing the same thing three times not ideal?

Reply:
It's not ideal, because of performance and readability (debugging). Therefore it's better to store it in a variable and then use the variable instead.
Thank you for mentioning it.